Document the impact of switching on/off full text search (#5917)
Particular/docs.particular.net: servicecontrol/audit-instances/creating-config-file.md

Default: `true`.

> NOTE: If the audit instance uses RavenDB 5 persistence (available starting 4.26.0), changing full-text search settings causes RavenDB indexes to be redeployed and rebuilt. Depending on the amount of documents stored in the database, the operation might take a long time and documents won't be available until it's completed.

Reviewer: ```suggestion
NOTE: If the audit instance uses RavenDB 5 persistence (available starting 4.26.0), changing the full-text search setting will cause indexes to be redeployed and rebuilt. Depending on the number of documents stored, this operation might take a long time and search results won't be available until completed.
```
